data_test.go

```diff
@@ -101,10 +102,15 @@ func WriteTestMortalityShapefile() {
 				geom.Point{X: -3999, Y: -3998},
 				geom.Point{X: -3999, Y: -3999},
 			}},
-			AllCause: 800.,
+			AllMort:    800.,
```

Comment:
I need to modify the test to make sure it actually tests that these new mortality rates are being handled properly.